src/redisearch_rs/inverted_index/src/tests.rs

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2006-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+*/
+
+use std::mem::ManuallyDrop;
+
+use ffi::t_docId;
+
+use crate::{RSAggregateResult, RSIndexResult, RSIndexResultData, RSResultType, RSResultTypeMask};
+
+#[unsafe(no_mangle)]
+pub extern "C" fn IndexResult_ConcatMetrics(
+    _parent: *mut RSIndexResult,
+    _child: *mut RSIndexResult,
+) {
+    // Do nothing since the code will call this
+}
+
+impl RSAggregateResult {
+    fn get(&self, index: usize) -> Option<&RSIndexResult> {
```

Comment:
Why not have this as proper public API?

Reply:
I just wanted to keep the public API as small as possible. But moved it here to have the tests in `/tests/` -> https://github.com/RediSearch/RediSearch/pull/6490/commits/e9640e7f0493fe063f61d386859f249d7d177902